src/sqlfluff/core/config.py

```diff
@@ -484,9 +515,35 @@ def from_kwargs(
             overrides["rules"] = ",".join(rules)
         return cls(overrides=overrides)
 
+    def get_templater(self, templater_name="jinja", **kwargs):
+        """Fetch a templater by name."""
+        templater_lookup = {
+            templater.name: templater
+            for templater in chain.from_iterable(
+                self._plugin_manager.hook.get_templaters()
+            )
```

Comment:
Teeny nitpick: It's tempting to put `if templater.name == templater_name` here, so the dictionary would at most have the one entry we want. But I think that's my OCD talking. Why bother?!?! Sorry... 🤪